### DISCUSSION TITLE
Does FreeTube integrate certain extensions ?

### DISCUSSION THREAD

**Discussion by Yakonche** [FreeTubeApp/FreeTube#2068] [Q&A]
Or, if it has not already integrated them, that they can be ?
I'm talking about these :
- [Return YouTube Dislike](https://github.com/Anarios/return-youtube-dislike)

Thank you in advance for your answers :)

Edit : I removed the SponsorBlock link, for obvious reasons x)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Sponsorblock is integrated into FreeTube and more of the their features will be added after the review process has been completed see #1849

Return YouTube Dislike has not been integrated. See https://github.com/FreeTubeApp/FreeTube/issues/1927#issuecomment-1007121970 for more info
